services/imagery/src/util.js

```diff
@@ -14,7 +15,12 @@ export async function convertPng(image) {
 
 /** Take EXIF data off of an image. */
 export async function removeExif(image) {
-  return await imageToBuffer(gm(image, 'image.jpg').noProfile());
+  await fs.writeFile('/tmp/temp.jpg', image);
```

Comment:
I think we should use the `tmp` library instead.

Reply:
Yeah that would be better, I think `tmp` also handles the deletion of the files as well when they're not needed anymore.

Reply:
also there's a race condition here where this fails if two images are having their exif data removed in overlapping intervals.